### PR TITLE
APIGOV-21192: Add missing CLI options and version introduced in

### PR DESCRIPTION
**Pull request by jquick-axway**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

**JIRA:**
https://jira.axway.com/browse/APIGOV-21192

**Summary:**
- Added missing CLI options such as `--account`, `--no-cache`, `--region`, etc.
- Indicated the CLI version each option was introduced in. (Except for options added <= v1.0.0).
- Specified what `--region` can be set to. Either `US` or `EU`.

## Checklist for contributors

Before submitting this PR, please make sure:

* [x] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [x] You have signed the [Axway CLA](https://cla.axway.com/)
* [x] You have verified the technical accuracy of your change
* [x] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [x] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

### CODE REVIEW

**Review by agrakhov — APPROVED** [review]

File: content/en/docs/integrate_with_central/cli_central/cli_command_reference.md [hunk 1]
```diff
@@ -37,15 +37,17 @@ The following table describes the usage, options, and arguments for the `get` co
...
-|`-q,--query "<RSQL-formatted query>"`                          |RSQL-formatted query to search for filters that match specific parameters|
-|`--team <name\|guid>`                                           |The team name or guid to use |
-|`--title <title of resource>`                                  |Title of resource(s) to fetch. Includes partial match|
-|`--tag <tag>`                                                  |Tag of resource(s) to fetch. Exact match|
-|`--attribute <key=value>`                                      |Attribute in key=value pair format to filter by. Exact match|
-|`--no-cache`                                                    |Do not use cache when communicating with the server|
+|`-s,--scope=<name>`                                            |Scope name for scoped resources.<br/>*(Added: v1.17.0)*|
+|`-q,--query "<RSQL-formatted query>"`                          |RSQL-formatted query to search for filters that match specific parameters.<br/>*(Added: v1.23.0)*|
+|`--team <name\|guid>`                                           |The team name or guid to use.<br/>*(Added: v2.6.0)*|
```
Comment (cb1kenobi) [inline content/en/docs/integrate_with_central/cli_central/cli_command_reference.md:45]: I noticed in the preview site that the \ gets rendered. I thought the | needed to be escaped as to not break the table, but maybe it's not necessary when used in an inline code block?

Comment (cb1kenobi) [reply content/en/docs/integrate_with_central/cli_central/cli_command_reference.md:45]: I guess this was my fault. Can you fix it please?

Comment (jquick-axway) [reply content/en/docs/integrate_with_central/cli_central/cli_command_reference.md:45]: Yeah, I can see that...
https://docs.axway.com/bundle/amplify-central/page/docs/integrate_with_central/cli_central/cli_command_reference/index.html

I'll fix it now. Thanks for bringing it up.

Comment (jquick-axway) [reply content/en/docs/integrate_with_central/cli_central/cli_command_reference.md:45]: I'm testing out what my options are via another PR. What a pain.

**Comment by lbadenhop** [comment]
@jquick-axway  Please let me know when this is ready to merge.

File: content/en/docs/integrate_with_central/cli_central/cli_command_reference.md (hunk 1 above)
Comment (jquick-axway) [reply content/en/docs/integrate_with_central/cli_central/cli_command_reference.md]: I worked around it by escaping the pipe outside of the back tick code block.

Supposedly, updating to the newest Hugo version will fix this. This is something the docs team will look into later.

**Comment by jquick-axway** [comment]
@lbadenhop , this PR is done and can be merged.

File: content/en/docs/integrate_with_central/cli_central/cli_command_reference.md [hunk 2]
```diff
@@ -37,15 +37,17 @@ The following table describes the usage, options, and arguments for the `get` co
...
-|`-q,--query "<RSQL-formatted query>"`                          |RSQL-formatted query to search for filters that match specific parameters|
-|`--team <name\|guid>`                                           |The team name or guid to use |
-|`--title <title of resource>`                                  |Title of resource(s) to fetch. Includes partial match|
-|`--tag <tag>`                                                  |Tag of resource(s) to fetch. Exact match|
-|`--attribute <key=value>`                                      |Attribute in key=value pair format to filter by. Exact match|
-|`--no-cache`                                                    |Do not use cache when communicating with the server|
+|`-s,--scope=<name>`                                            |Scope name for scoped resources.<br/>*(Added: v1.17.0)*|
+|`-q,--query "<RSQL-formatted query>"`                          |RSQL-formatted query to search for filters that match specific parameters.<br/>*(Added: v1.23.0)*|
+|`--team <name`\|`guid>`                                        |The team name or guid to use.<br/>*(Added: v2.6.0)*|
```
Comment (cb1kenobi) [inline content/en/docs/integrate_with_central/cli_central/cli_command_reference.md:45]: Heh, that'll work

**Review by cb1kenobi — APPROVED** [review]
LGTM

**Pull request merged**